decidim-elections/app/cells/decidim/votings/content_blocks/landing_page/metrics_cell.rb

```diff
@@ -5,8 +5,12 @@ module Votings
     module ContentBlocks
       module LandingPage
         class MetricsCell < Decidim::ViewModel
-          def show
-            content_tag(:div, "VotingMetricsCell")
+          delegate :current_participatory_space, to: :controller
+
+          def metrics
+            nil
+
+            # @metrics ||= VotingsMetricChartsPresenter.new(participatory_process: current_participatory_space)
```

Comment:
Is this comment needed? 😊

Reply:
No, it is just a hint for when the metrics are available. 😁